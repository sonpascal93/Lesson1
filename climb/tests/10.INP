1000
-22 26 -72 -76 86 -40 -25 33 15 80 17 -37 69 -98 17 -12 29 -91 -12 -36 -70 65 4 77 96 -73 31 -74 -72 3 27 -29 14 21 -67 -86 54 41 -16 -17 76 -54 0 -80 -52 -3 83 -38 30 32 -57 -50 -58 79 85 -1 -10 -24 9 88 -83 77 20 82 32 -40 -30 36 -51 -28 -91 -64 -94 -87 -33 85 -91 7 70 -91 -60 -68 -27 40 77 72 51 -63 -72 47 -49 77 76 47 57 -43 63 -73 87 -77 90 98 0 -70 -33 16 -19 54 -80 -86 38 7 -68 51 38 -16 -60 32 75 49 8 -37 77 -29 61 16 -24 -71 79 98 -77 -13 -98 0 -71 -44 -78 27 79 -34 -23 33 -95 42 -81 -14 -23 -8 29 -67 -12 59 30 38 -75 -21 -8 68 -56 14 1 70 92 -97 19 33 18 -45 64 26 -15 25 -42 -59 38 -1 79 -40 -97 -59 -50 34 2 -43 -92 -15 38 40 -97 8 70 4 0 -98 -9 32 14 78 -39 81 -39 -33 -81 76 15 -24 -6 7 -59 -61 60 -66 -46 -72 -33 -71 -72 79 43 30 90 -51 0 -86 -96 8 -15 -69 -94 97 81 -80 -39 86 -48 -18 -61 -4 95 64 -99 -61 28 -3 -50 -1 78 -26 -29 58 85 -11 -73 69 0 -90 97 74 -86 -15 -87 -72 76 83 -13 71 -68 -35 88 -96 82 -29 -97 40 34 -36 -83 97 91 -76 -71 0 44 28 77 78 37 31 88 -55 -14 87 69 70 -36 -17 96 27 -14 -23 -53 7 -90 -36 32 83 61 40 46 49 -14 38 91 34 80 -68 14 -14 95 -38 -8 -60 39 55 68 -3 -26 -90 4 -74 -37 66 -63 33 -99 11 1 -21 -73 -11 26 -64 -67 46 95 -71 -41 -7 -52 -30 43 -24 -45 -11 86 74 -29 36 61 -62 -37 -16 9 97 -42 -82 33 25 83 38 23 94 -60 -62 56 -18 -45 7 20 -1 -72 -61 0 54 -71 -18 -62 -44 79 52 69 -78 48 74 -14 -31 -94 83 36 -67 42 35 -41 16 30 52 71 -90 17 -54 -96 -91 5 -59 66 7 -38 40 60 -54 30 79 -66 -93 68 -19 93 1 -12 79 -37 40 54 27 77 -89 42 -20 -51 44 33 77 65 -68 67 93 93 48 55 -86 18 38 -75 33 17 87 -49 -71 97 53 34 -62 75 -99 -19 14 30 -41 47 76 16 50 3 19 72 -72 -22 -14 50 87 -5 78 -62 -18 -24 46 -33 -16 -91 3 98 89 -50 -12 -78 19 50 80 -85 17 -50 75 75 23 54 -44 11 59 19 58 -20 -58 97 -26 -67 97 -7 -99 14 25 -79 -65 80 -56 36 -22 -91 53 -74 26 -72 44 65 -90 -15 31 94 -96 96 -6 -57 -77 -39 72 -49 -22 57 60 -68 -23 73 -5 44 31 16 54 -46 35 -73 39 39 -98 9 -34 34 -44 -8 -95 16 39 55 68 -16 39 -59 -61 -35 -40 41 10 65 -99 40 0 6 -88 25 32 -22 -43 -32 0 41 34 27 83 -4 -40 -78 -77 -59 86 -98 85 96 97 -81 38 74 -6 -73 35 -44 57 29 -34 -97 -73 -43 -20 -19 -26 -96 8 93 -82 -11 0 16 73 -62 99 19 -64 23 50 46 54 -56 98 38 -26 31 3 0 47 -39 -20 34 -41 26 5 -77 -9 -73 -83 26 -8 91 16 -52 -44 51 29 -68 54 -73 1 -61 -26 20 -89 -16 95 -39 -66 -9 -32 -83 -81 -20 -28 -43 -80 -21 71 60 98 16 62 -85 39 50 79 -49 -81 -34 -18 44 64 -93 -50 -89 -75 -79 86 3 65 -99 60 92 93 -35 -54 -80 98 90 4 -64 -76 -64 11 -92 33 61 -17 12 -93 58 -34 48 -78 39 76 0 37 79 -57 49 -15 15 96 78 -48 -15 59 -76 56 -8 84 45 -53 12 39 -29 75 22 14 -80 -80 99 33 82 -68 -13 -10 -4 69 -24 -82 16 56 -84 55 -41 59 -45 -96 -16 37 47 8 80 86 -58 -79 -27 -60 51 42 -74 -78 -33 92 21 40 81 68 -69 -54 67 56 64 5 -50 -1 -93 -49 -56 68 -37 73 -32 50 96 45 -50 45 -1 -32 -86 58 -1 4 -85 -41 97 20 58 52 -79 53 47 84 2 32 -42 14 -67 -94 0 -31 -28 81 -1 -73 1 -88 14 7 22 87 -78 69 -50 24 30 -94 -87 -79 -77 48 -3 -15 84 92 -17 59 35 27 -74 -54 -77 -22 24 -24 -80 18 91 -18 0 81 35 2 95 -63 -91 -92 -60 10 -85 -10 -84 91 -60 -69 50 -23 53 54 -25 -86 -14 -38 72 -27 -46 86 -41 -47 6 24 35 -65 74 -71 -26 12 82 -8 51 -5 -5 -20 45 33 50 -94 66 -33 -66 -35 -88 -25 99 -73 -16 -84 96 -31 -6 2 -75 -45 -99 -45 -22 -9 -6 -9 -62 22 -95 40 43 -96 76 11 -57 -23 -70 30 -64 -76 20 -56 -62 99 -29 -48 96 44 17 14 73 0 -56 -52 11 -41 -19 25 19 36 -69 26 41 17 -7 -7 86 -34 -17 -43 40 -60 98 45 14 -12 