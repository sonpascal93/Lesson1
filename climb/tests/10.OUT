88
14 18 28 29 35 42 45 69 91 96 122 124 127 141 147 156 171 186 227 260 291 299 311 318 340 354 380 383 413 416 466 471 474 496 509 520 530 536 554 558 560 561 570 574 583 593 637 646 663 666 687 690 700 708 711 712 716 717 737 743 756 768 798 801 803 822 829 830 833 836 838 841 845 863 864 867 874 876 901 904 907 910 914 916 936 938 953 956 